InscryptionCommunityPatch/Card/Part3StatIcons.cs

```diff
@@ -0,0 +1,61 @@
+using DiskCardGame;
+using HarmonyLib;
+using UnityEngine;
+
+namespace InscryptionCommunityPatch.Card;
+
+[HarmonyPatch]
+internal static class Part3StatIcons
+{
+    [HarmonyPatch(typeof(DiskCardGame.Card), "RenderCard")]
+    [HarmonyPrefix]
+    private static void UpdateLiveRenderedCard(ref DiskCardGame.Card __instance)
+    {
+        CardStatIcons icons = __instance.gameObject.GetComponentInChildren<CardStatIcons>();
+        icons.attackIconRenderer.gameObject.transform.localPosition = new(0.39f, 0.033f, 0f);
+        icons.attackIconRenderer.gameObject.transform.localScale = new (0.2f, 0.2f, 1f);
```

Comment:
I would also make this and the localPosition Vector3 `private static readonly Vector3` objects in the class since they never change.

Reply:
Done - all the magic numbers/vectors are their own constants/readonlies